src/DragAndScale.ts

```diff
@@ -17,6 +17,18 @@ export class DragAndScale {
     dragging?: boolean
     viewport?: Rect
 
+    /**
```

Comment:
I don't think we need these hooks anymore as we can mark DragAndScaleState reactive on vue side. https://github.com/Comfy-Org/litegraph.js/pull/311